libminifi/include/c2/C2Payload.h

```diff
@@ -53,52 +54,45 @@ enum Direction {
   RECEIVE
 };
 
-class C2ContentResponse {
- public:
-  C2ContentResponse(Operation op);
-
-  C2ContentResponse(const C2ContentResponse &other);
-
-  C2ContentResponse(const C2ContentResponse &&other);
-
-  C2ContentResponse & operator=(const C2ContentResponse &&other);
-
-  C2ContentResponse & operator=(const C2ContentResponse &other);
-
-  inline bool operator==(const C2ContentResponse &rhs) const {
-    if (op != rhs.op)
-      return false;
-    if (required != rhs.required)
-      return false;
-    if (ident != rhs.ident)
-      return false;
-    if (name != rhs.name)
-      return false;
-    if (operation_arguments != rhs.operation_arguments)
-      return false;
-    return true;
+struct C2ContentResponse {
+  explicit C2ContentResponse(Operation op)
+      :op{ op }
+  {}
+
+  bool operator==(const C2ContentResponse &rhs) const {
+    return op == rhs.op
+        && required == rhs.required
+        && ident == rhs.ident
+        && name == rhs.name
+        && operation_arguments == rhs.operation_arguments;
   }
 
-  inline bool operator!=(const C2ContentResponse &rhs) const {
-    return !(*this == rhs);
-  }
+  bool operator!=(const C2ContentResponse &rhs) const { return !(*this == rhs); }
 
   Operation op;
   // determines if the operation is required
-  bool required;
+  bool required{ false };
   // identifier
   std::string ident;
   // delay before running
-  uint32_t delay;
+  uint32_t delay{ 0 };
   // max time before this response will no longer be honored.
-  uint64_t ttl;
+  uint64_t ttl{ std::numeric_limits<uint64_t>::max() };
```

Comment:
As we touch this code, can we also rename these to match the underscore ending member naming convention?

Reply:
We shouldn't, because these are public.

see "Struct Data Members" at https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
Fair point, didn't notice that it's a struct.